### PR TITLE
Adding deb-package-builder docker image

### LINKED ISSUE #34: Pre-compile the PHP binaries
It is too slow to compile multiple PHP versions for each Docker build. Can we pre-compile the PHP binaries, upload and download from GCS or somewhere?

### PR DESCRIPTION
**Pull request by tmatsuo**
This docker image is solely for creating debian packages that the php-nginx image will later download and use. That way, we'll be able to shorten the build time of the php-nginx itself significantly.

Fixes #34

### CODE REVIEW

**Comment by tmatsuo** [comment]
@dazuma I'd appreciate your review too

File: php-builder/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+# copy the gpgkeys again
+COPY gpgkeys /gpgkeys
+
+# Need to install debian packaging tools etc
+RUN apt-get update -y && \
+    apt-get install -y -q --no-install-recommends \
+        debhelper \
+        devscripts \
+        libparse-debcontrol-perl \
+	# headlers
```
Comment (dazuma) [inline php-builder/Dockerfile]: s/headlers/headers

Comment (tmatsuo) [inline php-builder/Dockerfile]: Done

**Comment by tmatsuo** [comment]
@dazuma Thanks for the review. I changed the directory name to `deb-package-builder`. PTAL

**Comment by tmatsuo** [comment]
I intentionally stopped the test to avoid hitting some quota behind the cloudbuild. I will re-run the test after a while.

**Pull request merged**